src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java

```diff
@@ -96,16 +96,11 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         jLabel1.setText("Custom lookup");
 
         dependentJLabel.setText("Dependent:");
-        dependentJLabel.setToolTipText("");
-
-        dependentJTextField.addActionListener(new java.awt.event.ActionListener() {
```

Comment:
Did you mean to remove this listener?

Reply:
Yes

Reply:
Okay... but why did you decide to remove it?